covid19/items/covid19_clinical_history

```diff
@@ -0,0 +1,110 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
+    "https://raw.githubusercontent.com/sensein/covid19/master/covid19/covid19_context"
```

Comment:
you are not using this context anywhere, so just reduce this list to the one url. this is true for all the items below.

Reply:
Reduce the list to this, right?
```
"@context": ["https://raw.githubusercontent.com/sensein/covid19/master/covid19/covid19_context"]

Reply:
it should reduce it to this to the other url

```
"@context": "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
```